### DISCUSSION TITLE
`brew list` top level items

### DISCUSSION THREAD

**Discussion by fishcharlie** [Homebrew/discussions#2373] [Everyday usage]
Is there a way to run `brew list` and **only** show items that were directly installed (ie. not installed due to a dependency)?

**Comment by cho-m** [comment]
Not that I know of via `brew list`.

There are some alternative ways discussed at #2346 

Another alternative, in addition to the ones shown in other discussion, is to use `brew bundle dump` (https://github.com/Homebrew/homebrew-bundle), which does this by [default](https://github.com/Homebrew/homebrew-bundle/blob/c6ede5d09ce03784bbd257352494d69d8449e1a5/lib/bundle/brew_dumper.rb#L55-L57):
